run-bft.sh

```diff
@@ -12,9 +12,16 @@ BEACON_COMMAND="cargo run -- start --nodisplay --verbosity 0 --dev 0 --beacon"
 echo "starting beacon as 0, check logs at ./beacon.log"
 $BEACON_COMMAND '' >beacon.log 2>&1 &
 
+# Start other validators without metrics
 for i in 1 2 3 4; do
-	VALIDATOR_COMMAND="cargo run -- start --nodisplay --verbosity 0 --dev ${i} --validator"
+	# Enable metrics for first validator only
+	extra_args=""
+	if [[ "$i" -eq 1 ]]; then
+		extra_args=" --metrics"
+	fi
+	VALIDATOR_COMMAND="cargo run -- start$extra_args --nodisplay --verbosity 0 --dev ${i} --validator"
```

Comment:
```suggestion
	VALIDATOR_COMMAND="cargo +stable run -- start$extra_args --nodisplay --verbosity 0 --dev ${i} --validator"
```
Just nice for those running `nightly` as their default rust 😇

Reply:
I suppose this also applies to the beacon, though we'll be getting rid of that shortly.

Reply:
Thanks, added it to both 👍